tests/sentry/event_manager/test_event_manager.py

```diff
@@ -2147,6 +2149,7 @@ def test_legacy_attributes_moved(self) -> None:
 
     @freeze_time()
     def test_save_issueless_event(self) -> None:
+        timestamp = before_now(minutes=5).isoformat()
```

Comment:
technically this one wasn't flaky (due to the freeze_time call) but if someone copy-pasted this test without the decorator it would be